src/handlers/har/preload.ts

```diff
@@ -0,0 +1,26 @@
+import { ipcRenderer } from 'electron'
+
+import { HarWithOptionalResponse } from '@/types/har'
+
+import { HarHandler } from './types'
+
+export function saveFile(data: HarWithOptionalResponse, prefix: string) {
+  return ipcRenderer.invoke(
+    HarHandler.SaveFile,
```

Comment:
Same thing here:

- moved functions out of `preload.ts`
- replaced the hardcoded channel name with an enum